src/desktop/apps/consign/client/analytics_middleware.ts

```diff
@@ -1,24 +1,13 @@
 import * as actions from "./actions"
 import { data as sd } from "sharify"
 
-const _analyticsHooks = require("../../../lib/analytics_hooks.coffee")
-
-// FIXME: Rewire
-let analyticsHooks = _analyticsHooks
-
 const analyticsMiddleware = store => next => action => {
   const result = next(action)
   const nextState = store.getState()
 
   // track certain types of actions
   switch (action.type) {
```

Comment:
This action may be safe to deprecate.

Reply:
@eessex - Mind elaborating a bit on the "may"? 😄

Reply:
i didn't want to touch the consign code too much, but i'm assuming that firing the analytics event is the only reason this action exists.

Reply:
We have a few actions we're tracking -- SUBMISSION_ERROR, SUBMISSION_CREATED , SUBMISSION_COMPLETED, etc. Do you mean all of these can be deprecated?

Reply:
Sorry placement of comment was unclear, `UPDATE_USER` is the action that I believe can be deprecated.